MonkeyLoader.Resonite.Integration/UI/Inspectors/CustomInspectorInjector.cs

```diff
@@ -17,61 +19,136 @@ internal sealed class CustomInspectorInjector
     {
         public override bool CanBeDisabled => true;
 
-        [HarmonyPrefix]
-        private static bool BuildUIForComponentPrefix(WorkerInspector __instance, Worker worker,
-            bool allowRemove, bool allowDuplicate, bool allowContainer, Predicate<ISyncMember> memberFilter)
-        {
-            if (!Enabled)
-                return true;
+        private static MethodInfo _buildHeaderMethod = AccessTools.Method(typeof(CustomInspectorInjector), nameof(OnBuildInspectorHeader));
+        private static MethodInfo _buildHeaderTextMethod = AccessTools.Method(typeof(CustomInspectorInjector), nameof(OnBuildInspectorHeaderText));
+        private static MethodInfo _buildBodyMethod = AccessTools.Method(typeof(CustomInspectorInjector), nameof(OnBuildInspectorBody));
+        private static MethodInfo _getEnabledMethod = AccessTools.Method(typeof(CustomInspectorInjector), nameof(GetEnabled));
 
-            var ui = new UIBuilder(__instance.Slot);
-            RadiantUI_Constants.SetupEditorStyle(ui);
-            ui.Style.RequireLockInToPress = true;
-            var vertical = ui.VerticalLayout(6f);
+        private static bool GetEnabled()
+        {
+            return Enabled;
+        }
 
-            if (worker is not Slot)
+        private static Label? GetNextBranchLabel(CodeInstruction[] codes, int startIndex)
+        {
+            int i = startIndex;
+            while (i < codes.Length)
             {
-                ui.Style.MinHeight = 32f;
-                ui.HorizontalLayout(4f);
-                ui.Style.MinHeight = 24f;
-                ui.Style.FlexibleWidth = 1000f;
+                if (codes[i].Branches(out var label))
+                {
+                    return label;
+                }
+                i++;
```

Comment:
Could use the post-increment in the indexer access too, for compactness. Can remove the {} on the if too